framework/src/org/checkerframework/common/value/ValueTransfer.java

```diff
@@ -925,7 +927,7 @@ private Range calculateRangeUnaryOp(
                 break;
             case GREATER_THAN_EQ:
                 thenRightRange = rightRange.refineLessThanEq(leftRange);
-                thenLeftRange = leftRange.refineGreaterThan(rightRange);
+                thenLeftRange = leftRange.refineGreaterThanEq(rightRange);
```

Comment:
Yes, this is just a bug. I probably should have PR'd it separately; I can do that if you'd like.